src/Audio/audioEngine.ts

```diff
@@ -7,8 +7,20 @@ import { Engine } from "../Engines/engine";
 import { IAudioEngine } from './Interfaces/IAudioEngine';
 import { DomManagement } from "../Misc/domManagement";
 
+export interface AudioEngineOptions {
```

Comment:
do you mind moving it to a separate file

Reply:
Split that off into its own class and renamed it IAudioEngineOptions.